h2/src/test/org/h2/test/store/TestMVStoreBenchmark.java

```diff
@@ -86,7 +86,7 @@ private void testMemoryUsageComparison() {
         hash = getMemory() - mem;
         mapList.size();
 
-        mapList = new ArrayList<>();
+        mapList = new ArrayList<>(count);
```

Comment:
`mapList.clear()` should be better. Also `mapList.size()` is not used.

The same below.

BTW, the whole list can be replaced with plain array or can be removed at all and replaced with passing of information about required constructor to `addEntries()`. But there is no reason to optimize test too much now, it can be done after switching to Java 8 with its method references and other useful stuff.